src/humanpy/hum_kin2.py

```diff
@@ -0,0 +1,298 @@
+PACKAGE = 'humanpy'
+import numpy
+import rospy
+import logging
+from tf import transformations
+import humanpy
+
+BASE_FRAME = '/map'
+KIN_FRAME = '/head/skel_depth_frame'
+KIN_FRAME2 = '/head/skel_depth_frame2'
+
+logger = logging.getLogger('humanpy')
+logger.setLevel(logging.INFO)
+
+class Orhuman(object):
+    def __init__(self, id, env, enable_legs=True, segway_sim=True):
+        assert id != ''
+        self.id = id
+        self.enabled = True
+        self.enable_legs = enable_legs
+        self.env = env
+        self.robot = env.GetRobot('herb')
+        _,self.body = humanpy.initialize(sim=True, user_id=id, env=env)
+        self.body.Enable(False)
+        self.initialDOFvalues = self.body.GetDOFValues()
+        self.currentDOFvalues = self.body.GetDOFValues()        
+        self.starting_angle = dict()
+        self.last_updated = rospy.get_rostime().secs 
+        if segway_sim==True: 
+            self.refsys = KIN_FRAME2 
+        else: 
+            self.refsys = KIN_FRAME
```

Comment:
...still seems to be there? 
https://github.com/personalrobotics/humanpy/blob/feature/humanpy/src/humanpy/humantracking_kinect2.py#L23